src/simple_graph.py

```diff
@@ -82,3 +82,45 @@ def adjacent(self, n1, n2):
                 return False
         except KeyError:
             raise KeyError
+
+    def dft(self, start):
+        """Return a list of visited nodes for depth first traversal."""
+        to_visit = [start]
+        visited = []
+        while to_visit:
+            next_val = to_visit.pop()
+            if next_val not in visited:
+                visited.append(next_val)
+                to_visit = to_visit + self.graph[next_val]
+        return visited
+
+    def bft(self, start):
+        """Return a list of visited nodes for breadth first traversal."""
+        to_visit = [start]
+        visited = []
+        while to_visit:
+            next_val = to_visit.pop()
+            if next_val not in visited:
+                visited.append(next_val)
+                to_visit = self.graph[next_val] + to_visit
```

Comment:
Not docking points here, just noting that you could use the Queue object you created.